src/user/event_service.rst

```diff
@@ -0,0 +1,141 @@
+.. _event_service:
+
+EventService
+============
+
+Workflow
+------------
+
+Here is the structure of workflow.
+
+.. image:: ../_images/work_concepts.png
+   :width: 6.5in
+   :height: 2.66667in
+
+- **Workflow**. One workfow corresponds to a scientifc objective defning intermediate steps for the user
+  to accomplish. A workfow comprises a group of constituent tasks and their topological relationship.
+
+- **Task**. A task is a software program or application that utilizes computing resources to achieve one
+  of the intermediate steps in a workfow. It normally processes a set of data.
+
+- **Job**. A job is a PanDA object with distinct states that corresponds a sub-unit of a task. A single
+  task consists of multiple jobs, and each job is executed by a worker, which is an abstraction of the
+  execution point, such as a worker node and CPU cluster. A job can process a list of files, a file or
+  a part of one file. In ATLAS experiment, one file normally stores many ordered events (particle objects).
+  One job can process a part of the jobs, for example 0~1000 events for the first job and 1000~2000 for
+  another job. In the job level, the events are processed in order in one sequence. If one of the fails,
+  the job will fail.
```

Comment:
If one of the events fails?
This is still confusing how this translates to LSST world.

Also, is PanDA job the unit that a Pilot runs?

Reply:
updated this part to:  If one of the events fails, the job will fail (Generally EventService is not enabled. Job is the basic unit for bookkeeping. Without event level bookkeeping, all events need to be processed in order and an event failure will fail the job). In addition, a PanDA jo is the basic unit that a pilot runs.